### DISCUSSION TITLE
Discusssion aber keine Issues?

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [willuhn/hibiscus#134] [Entwickler-Diskussion]
Vielen Dank für diese Opensource Software. Mit zur Zeit 69 Forks, 222 Stars und 26 contributors sieht das Projekt ja ganz gut aus.
Verwundert bin ich dafür dass "Issues" nicht aktiviert ist. Wie werden Probleme und Feature-Requests für diese Software gehandhabt?

**Comment by willuhn** [comment]
Hallo,

du kannst mir die einfach per Mail schicken. Ich habe die Issues aus verschiedenen Gründen nicht aktiviert. Ich möchte neben dem Forum unter https://homebanking-hilfe.de/forum/index.php?f=33 und Mail nicht noch einen weiteren Feedback-Kanal haben. Die Liste der offenen Issues würde im Laufe der Zeit immer länger werden. Ich hatte vor einiger Zeit auf willuhn.de mal ein Bugzilla betrieben. Dort war das genauso. Irgendwann hatte ich dann den Überblick verloren. Diese immer länger werdende Liste von offenen Aufgaben hatte mich zunehmend gestresst (ok, bei Bugzilla kam noch hinzu, dass es dauernd voll gespamt wurde). Ich entwickle Hibiscus im Wesentlichen allein und in meiner Freizeit. Ich bin nicht mehr der Jüngste und ich habe Familie. Andere können sicher besser mit offenen Issues umgehen. Mich persönlich stresst es. Das möchte ich in einem Freizeit-Projekt nicht. Weil ich das auch die nächsten Jahrzehnte noch gern machen möchte.

Gruss
Olaf
